ui/StatusQ/src/StatusQ/Popups/Dialog/+qt6/StatusFileDialog.qml

```diff
@@ -0,0 +1,37 @@
+import QtQuick 2.15
+import QtQuick.Dialogs
+
+Item {
+    id: root
+
+    property alias title: dlg.title
+    property alias nameFilters: dlg.nameFilters
+    property alias selectedFile: dlg.selectedFile
+    property alias selectedFiles: dlg.selectedFiles
+    property bool selectMultiple
+
+    property alias modality: dlg.modality
+    property alias currentFolder: dlg.currentFolder
+
+    property string picturesShortcut: ""
```

Comment:
This is `readonly` in the other version

Reply:
You can point to `StandardPaths.writableLocation(StandardPaths.PicturesLocation)` to get the same behavior (with `import Qt.labs.platform 1.1`) as with the Qt5 variant

Reply:
Edit:  `import QtCore` as this is Qt6